.travis.yml

```diff
@@ -8,13 +8,13 @@ arch:
 
   packages:
     - python
-    - python-nose
+    - python-pytest
```

Comment:
I wonder if we'd like to have any dependency here at all. Can't we use `python -m unittest` or use a very small test aggregator script?

I'm also down for moving to pytest later down the line if that's the plan...

Reply:
My other PR makes extensive use of pytest for coverage and pytest allows filtering and dropping into a pdb shell. It has so many advantages.

Reply:
Ah yeah, so disregard this please :+1: